app/views/email_signups/new.html.erb

```diff
@@ -1,14 +1,29 @@
-<% page_title "Email alert subscription" %>
+<% page_title t("email_signups.new.title") %>
 <% page_class "email-signup-show" %>
 
 <header class="block email-header">
   <div class="inner-block">
-    <h1>Email alert subscription</h1>
+    <h1 class="govuk-heading-l"><%= t("email_signups.new.title") %></h1>
   </div>
 </header>
 
 <div class="block">
   <div class="inner-block">
-    <%= render 'form', email_signup: @email_signup %>
```

Comment:
Can this partial be deleted?

Reply:
Oops - yes, I've done that in the second commit 👍.